src/testing/docker/default-docker-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This file contains the same content as `src/testing/docker/travis_docker_test.sh`